### PR TITLE
Reader: Adjust padding around Longreads publisher intros

### PR DESCRIPTION
**Pull request by kjellr**
We sometimes include short introduction block quotes that highlight publishers on Longreads posts. When we do, they always have the `publisher-intro` class. 

The Reader adds some unnecessary padding around the images within those intros. This PR removes that extra padding. 

---

**Before:** (Changes are evident in the spacing around the "TMI" logo in these screenshots)

screen shot 2017-05-31 at 8 40 00 am
screen shot 2017-05-31 at 8 40 14 am

**After:**

screen shot 2017-05-31 at 8 39 32 am
screen shot 2017-05-31 at 8 39 10 am

---

I used the blog-specific class for Longreads (`blog-70135762`) to target this. We’ve done similar things for the Daily Post and Discover in the past. 

To test, visit full-post views of stories with the publisher intro. For example: 
http://calypso.localhost:3000/read/feeds/22973954/posts/1476030312
http://calypso.localhost:3000/read/feeds/22973954/posts/1433727023

### CODE REVIEW

**Comment by jancavan** [comment]
Thanks @kjellr. Should the text still wrap around the image at <660?

File: client/blocks/reader-full-post/_content.scss [hunk 1]
```diff
@@ -331,6 +331,22 @@
 	}
 }
 
+// Longreads-Specific Full Post View Styles
+.blog-70135762 .reader-full-post__story-content {
+	
+	.publisher-intro {
+		img {
```
Comment (jancavan) [inline client/blocks/reader-full-post/_content.scss]: Small nitpick. Probably doesn't need to be nested? `.publisher-intro img`

Comment (kjellr) [inline client/blocks/reader-full-post/_content.scss]: Sure, I can do that. I'll also need to un-nest the other change on L342 then: `.publisher-intro p:first-child`.

**Comment by kjellr** [comment]
> Thanks @kjellr. Should the text still wrap around the image at <660?

Yep — we wrap them on the Longreads site too:

screen shot 2017-05-31 at 1 08 37 pm

**Comment by kjellr** [comment]
@jancavan the nesting should be all set. Would you mind taking another look?

**Comment by jancavan** [comment]
> Yep — we wrap them on the Longreads site too

Cool. Fixed in https://github.com/Automattic/wp-calypso/pull/14626/commits/65384831d32d1c61e39503847509c193965417ab.

I think this should be good to go 👍

**Comment by kjellr** [comment]
Looks great, thanks @jancavan!

**Pull request merged**